src/main/scala/com/vngrs/etl/configs/Configs.scala

```diff
@@ -0,0 +1,6 @@
+package com.vngrs.etl.configs
+
+/**
+  * Generic configuration interface
+  */
+trait Configs
```

Comment:
What is the purpose of that interface? Is it only to limit upper type of typeclass?